stages/org.osbuild.dnf

```diff
@@ -234,12 +234,15 @@ def main(tree, options):
 
     # verify metadata checksum
     for repoid, repo in enumerate(repos):
-        algorithm, checksum = repo["checksum"].split(":")
+        algorithm, expected_checksum = repo["checksum"].split(":")
         assert algorithm == "sha256"
         cachedir = dnf_cachedir(f"repo{repoid}", repo, releasever, basearch)
         with open(f"{tree}/var/cache/dnf/{cachedir}/repodata/repomd.xml", "rb") as f:
             repomd = f.read()
-        assert hashlib.sha256(repomd).hexdigest() == checksum
+        checksum = hashlib.sha256(repomd).hexdigest()
+        if checksum != expected_checksum:
+            print(f"expected repository with checksum {expected_checksum}, got {checksum}")
```

Comment:
Could you also include which repository it was? We don't have repo ids, but the index into the repo array will already help.

Reply:
I thought having the checksum would allow you to figure it out, but I guess the index is a good idea. Will add.

Reply:
Thanks! `repo1` is easier to read than comparing checksums :)